backend/users/views/team/user_role.py

```diff
@@ -0,0 +1,106 @@
+import json
+from typing import List
+
+from django.conf import settings
+from django.contrib.auth.decorators import login_required
+from django.core.exceptions import PermissionDenied
+from django.http import HttpRequest
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+from django.views.decorators.cache import cache_control
+from inertia import inertia
+from pydantic import BaseModel
+
+from paul.views.data_model import Breadcrumb, serialize_page_props_decorator
+from users.forms import ChangeRoleForm
+from users.models import RoleChoices
+from users.views.team.user import (
+    PAGE_TABS,
+    UserPageProps,
+    get_base_url,
+    get_breadcrumbs,
+    get_description,
+    get_requested_user,
+    get_tabs,
+    get_title,
+)
+
+
+class RoleChoicesModel(BaseModel):
+    value: str
+    label: str
+    disabled: bool = False
+    description: str = ""
+
+
+class UserRolePageProps(UserPageProps):
+    roles: list[RoleChoicesModel]
+    userRole: str
+
+
+@login_required
+@cache_control(private=True)
+@inertia("users/team-user/role")
+@serialize_page_props_decorator
+def manage_user_role(request: HttpRequest, user_id: str) -> UserRolePageProps:
+    """
+    Redirect to manage_user view for user role
+    """
+    user = get_requested_user(user_id=user_id)
+
+    breadcrumbs = get_breadcrumbs(user)
+    breadcrumbs.append(
+        Breadcrumb(
+            label=str(_("Activity Log")), url=reverse("users:manage-user-activity-log", kwargs={"user_id": user_id})
+        )
+    )
+
+    errors = None
+    if request.method == "POST":
+        if user.main_role in (RoleChoices.SUPER_ADMIN, RoleChoices.SUPPORT_ADMIN):
+            raise PermissionDenied(_("You cannot change the role of this user."))
+
+        form = ChangeRoleForm(json.loads(request.body), instance=user)
+        if not form.is_valid():
+            errors = dict(form.errors)
```

Comment:
We should namespace the `errors` in order to prevent name clashes on pages with several forms.